shared/src/commonMain/kotlin/com/twilio/passkeys/exception/TwilioException.kt

```diff
@@ -16,23 +16,83 @@
 
 package com.twilio.passkeys.exception
 
-internal const val INVALID_JSON_PAYLOAD_ERROR = "INVALID_JSON_PAYLOAD"
-internal const val UNKNOWN_ERROR = "UNKNOWN"
+internal const val DOM_EXCEPTION = 1001
+internal const val USER_CANCELED_EXCEPTION = 1002
+internal const val INTERRUPTED_EXCEPTION = 1003
+internal const val UNSUPPORTED_EXCEPTION = 1004
+internal const val NO_CREDENTIAL_EXCEPTION = 1005
+internal const val MISSING_ATTESTATION_OBJECT_EXCEPTION = 1006
+internal const val INVALID_JSON_PAYLOAD_EXCEPTION = 1008
```

Comment:
why not `1007`?

Reply:
Updated. Thanks for catching this, I totally missed it